image-extractor.php

```diff
@@ -12,7 +12,8 @@
 $final_response = array();
 $images = array();
 
-if(isset($_GET['url'])){
+if(isset($_GET['url']))
+{
```

Comment:
I don't think there is any need of putting the brackets in a new line
They unnecessarily increase the length of file